lnbits/static/js/wallet.js

```diff
@@ -1112,15 +1112,7 @@ window.WalletPageLogic = {
     }
   },
   watch: {
-    'g.wallet.id'(newVal, oldVal) {
-      try {
-        this.fetchChartData()
```

Comment:
is the chart data. still updated when switching wallets?
why was this removed, does not look related to the original issue?

Reply:
strangelly it does!

Reply:
i'll test a bit more, i can add it back in!

Reply:
if its not related to this PRs fix then please do